build/jest-framework-shims.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-env node */
+global.requestAnimationFrame = callback => {
```

Comment:
What is this for?

Reply:
Required for running react tests in node - they ask you to bring your own polyfills.